### PR TITLE
Add new EdgeDeployment arguments

### PR DESCRIPTION
**Pull request by shawnps**
Original PR from @vvuksan is [here](https://github.com/signalsciences/go-sigsci/pull/50)

This adds my commit which updates `ActivateVersion` to a pointer.

> Add support for PercentEnabled and ActivateVersion arguments to the edgeDeployment call.

### CODE REVIEW

**Review by vvuksan — DISMISSED** [review]
:+1:

File: api.go [hunk 1]
```diff
@@ -2901,14 +2901,24 @@ func (sc *Client) DeleteEdgeDeployment(corpName, siteName string) error {
...
+type CreateOrUpdateEdgeDeploymentServiceBody struct {
+	ActivateVersion *bool `json:"activateVersion,omitempty"` //Activate Fastly VCL service after clone
+	PercentEnabled  int   `json:"percentEnabled,omitempty"`  //Percentage of traffic to send to NGWAF@Edge
+}
+
 // CreateOrUpdateEdgeDeploymentService copies the backends from the Fastly service to the
 // Edge Deployment and pre-configures the Fastly service with an edge dictionary and custom VCL.
-func (sc *Client) CreateOrUpdateEdgeDeploymentService(corpName, siteName, fastlySID string) error {
+func (sc *Client) CreateOrUpdateEdgeDeploymentService(corpName, siteName, fastlySID string, body CreateOrUpdateEdgeDeploymentServiceBody) error {
+	b, err := json.Marshal(body)
```
Comment (freeformz) [inline api.go:2916]: Why do this before the `sc.fastlyKey` check below?

**Review by freeformz — DISMISSED** [review]

File: api.go (hunk 1 above)
Comment (vvuksan) [reply api.go:2916]: Fair point :-)

Comment (shawnps) [reply api.go:2916]: @freeformz thanks -- fixed

**Review by freeformz — APPROVED** [review]

**Pull request merged**